flytekit/remote/remote.py

```diff
@@ -2080,6 +2087,7 @@ def execute_local_launch_plan(
         domain = resolved_identifiers.domain
         try:
             flyte_launchplan: FlyteLaunchPlan = self.fetch_launch_plan(**resolved_identifiers_dict)
+            flyte_launchplan._python_interface = entity.python_interface
```

Comment:
When a function adjusts another objects private attribute, it feels like an anti-pattern. 

I'll be okay with either:

1. `RemoteEntity._python_interface` to `RemoteEntity.python_interface`
2. Adding a `setter`:

```python
class RemoteEntity:
    @python_interface.setter
    def python_interface(self, value):
        self._python_interface = value
```

This way, we can write `flyte_launchplan.python_interface = entity.python_interface` here.

Reply:
ok will do

Reply:
I actually completely agree with you, I hate modifying private properties. Sadly this was the previous case. But I now made sure and do not see it used anywhere, so updated it everywhere.